src/sentry/static/sentry/app/views/organizationIntegrations/integrationItem.tsx

```diff
@@ -7,15 +7,20 @@ import {t} from 'app/locale';
 import IntegrationIcon from 'app/views/organizationIntegrations/integrationIcon';
 import Tooltip from 'app/components/tooltip';
 import space from 'app/styles/space';
+import {Integration} from 'app/types';
 
-export default class IntegrationItem extends React.Component {
+type Props = {
+  integration: Integration;
+  compact?: boolean;
+};
+export default class IntegrationItem extends React.Component<Props> {
   static propTypes = {
     integration: PropTypes.object.isRequired,
     compact: PropTypes.bool,
   };
 
   render() {
-    const {integration, compact} = this.props;
+    const {integration, compact = false} = this.props;
```

Comment:
@nolachen Probably better to just set the defaultValue of compact to false rather than do this in a render method.